scripts/codePush.js

```diff
@@ -7,27 +7,43 @@ const childProcess = require('child_process');
 
 const packageJson = require('../package.json');
 
+const isRunningForAncillaries = process.argv.slice(2)[0] === '--ancillaries';
+
 const exec = (command, options) =>
   childProcess.execSync(command, {
     stdio: 'inherit',
     ...options,
   });
 
-const targetBinaryVersion =
+let targetBinaryVersion =
   packageJson.rnkiwimobile['code-push-target-binary-version'];
 
-const entryFile = path.join(__dirname, '..', 'app/native.js');
+let entryFile = path.join(__dirname, '..', 'app/native.js');
+
+let targetProjectAndroid = 'Kiwicom/mobile-android';
+let targetProjectIOS = 'Kiwicom/mobile-ios';
+let bundleName = 'main.jsbundle';
+
+// If --ancillaries flag is passed override entry file, target binary version and target project
+if (isRunningForAncillaries) {
+  entryFile = path.join(__dirname, '../app/ancillaries-native.js');
+  targetBinaryVersion =
+    packageJson.rnkiwimobile['code-push-ancillaries-target-binary-version'];
+  targetProjectAndroid = 'Kiwicom/mobile-android-ancillaries';
+  targetProjectIOS = 'Kiwicom/mobile-ios-ancillaries';
+  bundleName = 'ancillaries.jsbundle';
+}
 
 function processCommand() {
   const target = 'Staging';
   console.log(`Building for ${target}...`);
 
   exec(
-    `appcenter codepush release-react -a Kiwicom/mobile-android --entry-file ${entryFile} --target-binary-version "${targetBinaryVersion}" -d ${target}`,
+    `appcenter codepush release-react -a ${targetProjectAndroid} --entry-file ${entryFile} --target-binary-version "${targetBinaryVersion}" -d ${target} --bundle-name ${bundleName}`,
   );
 
   exec(
-    `appcenter codepush release-react -a Kiwicom/mobile-ios --entry-file ${entryFile} --target-binary-version "${targetBinaryVersion}" -d ${target}`,
+    `appcenter codepush release-react -a ${targetProjectIOS} --entry-file ${entryFile} --target-binary-version "${targetBinaryVersion}" -d ${target} --bundle-name ${bundleName}`,
   );
 }
```

Comment:
Why these changes?

Reply:
We want to have separate CodePush release process for ancillaries.

Reply:
We discussed this before with callstack and native team. We do not want separate bundles. This means duplicated code and bloated app size

Reply:
@tbergq We have added this because we can't use `RNKiwiSharedBridge` and if we are going to use the same bundle with separated bridge instances:
1. Each bridge have a larger bundle to load (slower first render)
2. We have to plan updates and releases together. (`master` should be always production ready, which is not aligned with current workflow)
3. Other thing is that native team said that they don't like current architecture where they have to know to much RN details in their app. e.g. it doesn't make sense for native mobile app to have `codepush` as dependency or if they want to use some view they have to know about how our bridge is working.

So, if we add it like this as a separate bundle we are just adding 3MB to the native app. That can be even smaller if we are going to use `unbundle` as a bundle command which is code splitting for React native and it results in smaller bundles and faster loads.

Maybe at some point, we can join again together, but that would require changes in our architecture.

Reply:
Well, then I suggest you create a new repo if you can not use our setup

Reply:
We can't create new repo because we don't want our native app to load multiple frameworks right? And we still need to use some of the features you've already made.
Try to think of this repo as a mobile framework, which it is, we just provide some views to our native app.
You are working on hotels we are working on ancillary views for MMB. Your views are the full screen which has different architecture requirements then our views which should be inline.